docs/tech-design/safety-pool.md

```diff
@@ -0,0 +1,86 @@
+# Safety Pool functionality for the Staking Pool
+
+| Specification | Implementation | Last revision |
+|:-----------:|:-----------:|:-------------:|
+| WIP         |  WIP        | 2022-05-13    |
+
+---
+
+**Specification ownership:** [Emily Martins]
+
+**Authors**:
+
+-   [Emily Martins]
+
+**Implementation ownership:** [Emily Martins]
+
+[Emily Martins]: https://github.com/emiflake
+
+[Jack Hodgkinson]: https://github.com/jhodgdev
+
+**Current status:** Initial draft in need of editing and review from [Jack Hodgkinson].
+
+---
+
+In order for Agora’s StakingPool to act as a *SafetyPool,* it needs to be able to support a workflow for slashing staked governance tokens to act as a safety mechanism. This document outlines the changes that Agora needs to receive in order to support this.
```

Comment:
Do `StakingPool` and `SafetyPool` exist as components anywhere in-code atm? If no, then use 'staking pool' and 'safety pool'. If yes, render them in monospace.